### ISSUE TITLE
Matches not Parseing in the correct order

### ISSUE THREAD

**Issue by Techno11** [wdavies973/TBA-API-V3#5]
The matches are being parsed in the wrong order, Match 2 is parsed right after Match 19, 3 after Match 29, so the parsed matches apear in the order:

1-19, 2, 20-29, 3, 30-39, 4, 40-49, 50, etc...

If possible please fix. Thanks!

**Comment by Techno11** [comment]
Also, if quarterfinal/semifinal/final matches could follow the last qualifacation mach rather then be before match one, that would be great as well.  Thanks!

**Comment by widavies** [comment]
Did you fix this in your pull request or is it still an issue?

**Comment by Techno11** [comment]
No.  It is still an issue

**Comment by Techno11** [comment]
Not Fixed in PR #7

**Comment by widavies** [comment]
Hey, just wanted to let you know I'll fix both of these issues within about a week, I'm currently busy finishing off some scouting software, but I'll have this repository up and running soon. Thanks!

**Comment by widavies** [comment]
Alright, this issue is fixed and added to the release at: [v1.0.1](https://github.com/wdavies973/TBA-API-V3/releases/tag/1.0.1). If you don't want to update the library, you can easily sort like so:

```
   TBA.setAuthToken("");
   SMatch[] matches = new TBA().getMatches("2017mndu2");
   Collections.sort(Arrays.asList(matches));
   for(SMatch m : matches) {
        System.out.println(m.getMatchNumber());
   }
```
